bind/files/debian/named.conf.logging

```diff
@@ -0,0 +1,120 @@
+// by default, make sure you add option ‘querylog no;’ - then you can toggle
+// query logging on (and off again) using command ‘rndc querylog’
+//
+        channel queries_log {
+                file "{{ map.log_dir }}/queries" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity info;
+        };
+        channel query-errors_log {
+                file "{{ map.log_dir }}/query-errors" versions {{salt['pillar.get']('bind:config:use_extensive_logging:versions', '3')}} size {{salt['pillar.get']('bind:config:use_extensive_logging:size', '20m')}};
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                severity dynamic; 
+        };
+        channel default_syslog {
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                syslog daemon;
+                severity info;
+        };
+        channel default_debug {
+                print-time yes;
+                print-category yes;
+                print-severity yes;
+                file "named.run";
+                severity dynamic;
+        };
+
+        category default { default_syslog; default_debug; default_log; };
+        category config { default_syslog; default_debug; default_log; }; 
+        category dispatch { default_syslog; default_debug; default_log; };
+        category network { default_syslog; default_debug; default_log; };
+        category general { default_syslog; default_debug; default_log; };
+        category resolver { auth_servers_log; default_debug; };
+        category cname { auth_servers_log; default_debug; };
+        category delegation-only { auth_servers_log; default_debug; };
+        category lame-servers { auth_servers_log; default_debug; };
+        category edns-disabled { auth_servers_log; default_debug; };
+        category dnssec { dnssec_log; default_debug; };
+        category notify { zone_transfers_log; default_debug; };
+        category xfer-in { zone_transfers_log; default_debug; };
+        category xfer-out { zone_transfers_log; default_debug; };
+        category update{ ddns_log; default_debug; };
+        category update-security { ddns_log; default_debug; };
+        category client{ client_security_log; default_debug; };
+        category security { client_security_log; default_debug; };
+        category rate-limit { rate_limiting_log; default_debug; };
+        category spill { rate_limiting_log; default_debug; };
+        category database { rate_limiting_log; default_debug; };
+        category rpz { rpz_log; default_debug; };
+        category queries { queries_log; };
+        category query-errors {query-errors_log; };
```

Comment:
Same as above, like

```yaml
bind:
  config:
    use_extensive_logging:
      category:
        default:
          - default_syslog
          - default_debug
          - default_log
        config:
          - default_syslog
          - default_debug
          - default_log
...
        queries:
          - queries_log
```
and then just join them as it's done [here](https://github.com/saltstack-formulas/bind-formula/blob/master/bind/files/redhat/named.conf.local#L38)

What do you think?